### DISCUSSION TITLE
Does renovate allow you to target the `support` tag of a monorepo?

### DISCUSSION THREAD

**Discussion by mannycarrera4** [renovatebot/renovate#22437] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

github enterprise

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have a library that support 3 major versions, current, support, and next major version. I want renovate to open PRS not only for the current major version, but also for the latest support version

### What you tried so far.

I tried followTag: "support" but not sure if that's what I want because I need renovate to follow stable as well as the support tag on npm. For reference, here's the package: https://www.npmjs.com/package/@workday/canvas-kit-react?activeTab=versions

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
If you need to follow a tag then followTag would be the only way, but I don't understand how you're using these so can't answer further

**Comment by mannycarrera4** [comment]
Thanks for replying so quickly!

I ended up using `baseBranches` where one of my branches points to a different version and renovate was able to pick it up!
